KostyantynTadlya/HW3/PracticalTask2.py

```diff
@@ -0,0 +1,9 @@
+a = input("Input 4 digit number")
```

Comment:
good job, 
look at another implementation
a = input("Input 4 digit number: ")
number = int(a)

digits_product = int(a[0]) * int(a[1]) * int(a[2]) * int(a[3])
print('Digits product =', digits_product)

reversed_number = int(a[::-1])
print('Reverse number =', reversed_number)

ordered_digits = sorted(set(str(number)))
print('Ascending order of digits:', ordered_digits)